CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/compute.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 The Pontem Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Compute API service proxy."""
+import re
+
+import google.auth
+
+from google.cloud.pontem.sql.replicator.util import gcp_api_util
+
+COMPUTE_SERVICE = 'compute'
+COMPUTE_SERVICE_VERSION = 'v1'
+
+# RFC 1035 regex (https://www.ietf.org/rfc/rfc1035.txt) for firewall rule names
+RFC1035_REGEX = r'(?:[a-z]([-a-z0-9]*[a-z0-9])?)\Z'
+
+
+def build_compute_service(credentials=None):
+    """Authorizes and sets custom CloudSQL Replicator user agent.
```

Comment:
ditto to my comment above

Reply:
Changed to
"Builds an authorized compute service proxy with a custom user agent."